src/main/java/microsoft/exchange/webservices/data/EwsXmlReader.java

```diff
@@ -1121,4 +1121,17 @@ private static boolean isNullOrEmpty(String namespacePrefix) {
 
   }
 
+  /**
+   * Gets the error message which happened during {@link #readValue()}.
+   *
+   * @param details details message
+   * @return error message with details
+   */
+  private String getReadValueErrMsg(final String details) {
+    return String.format(
```

Comment:
I am not sure that it will improve performance, because error message is not typical case (it should happen very rare in positive cases), but I can change it if you want.